src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2402,12 +2402,14 @@ namespace wallet_rpc
     struct response_t
     {
       bool multisig;
+      bool kex_is_done;
       bool ready;
       uint32_t threshold;
       uint32_t total;
 
       BEGIN_KV_SERIALIZE_MAP()
         KV_SERIALIZE(multisig)
+        KV_SERIALIZE(kex_is_done)
         KV_SERIALIZE(ready)
```

Comment:
I would suggest this for best backwards compatibility 

```suggestion
        KV_SERIALIZE(ready)
        KV_SERIALIZE_OPT(kex_is_done, ready)
```

Reply:
Not for this codebase, but it would be it would be a problem for downstream code if they used `wallet_rpc_server_commands_defs.h`.

Reply:
I think we usually don't care about downstream code that use `src/wallet/wallet_rpc_server_commands_defs.h`, at least I'm not aware of any such projects that make use of it.

Reply:
Okay fine then I'll drop it